torch_xla/csrc/ir.h

```diff
@@ -138,6 +138,15 @@ class XlaNode : public torch::lazy::Node {
 
   std::string ToString() const override;
 
+  void MarkDynamicDimension(uint32_t dim) { dynamic_dims_.insert(dim); }
+
+  const std::unordered_set<uint32_t>& dynamic_dims() const {
+    return dynamic_dims_;
+  }
+
+ protected:
+  std::unordered_set<uint32_t> dynamic_dims_;
```

Comment:
nit `unbounded_dynamic_dims_ ` so we don't confuse with bounded dynamisim work that @vanbasten23 has been working on.

Reply:
Yes, it's indeed confusing, let me rename